pkgs/servers/akkoma/pleroma-fe/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitea, fetchYarnDeps
+, fixup_yarn_lock, yarn, nodejs
+, jpegoptim, oxipng, nodePackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pleroma-fe";
+  version = "2022-09-10";
+
+  src = fetchFromGitea {
+    domain = "akkoma.dev";
+    owner = "AkkomaGang";
+    repo = "pleroma-fe";
+    rev = "d7499a1f91fb5e4ec947445c566f8ae41cec6c72";
+    hash = "sha256-I+d+oTOTNaMPkk9vdFHhm/g8EtPTZnsWv8yPUa0uNHQ=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-UkSdVRqFp6bj7zz4ZRMD47tVpuS7x7F1MiEei+87kyg=";
+  };
```

Comment:
I think this isn’t necessary, although removing it may be an import-from-derivation